packages/next-app/src/contexts/PoolMetadataContext.tsx

```diff
@@ -1,11 +1,12 @@
 "use client";
-
 import { TypenameEnum } from "@balancer-pool-metadata/schema";
+import { isEqual } from "lodash";
```

Comment:
I'm not sure where tree-shaking is at this point, and I think lodash is a commonjs module and that webpack can only do tree shaking with ES6 modules(to be confirmed, but that's the last I remember), so to be safe would only import to avoid bloating the bundle size

```suggestion
import isEqual from "lodash/isEqual";
```

Reply:
I'll research about that, thanks 👍